include/swift/AST/Builtins.def

```diff
@@ -723,6 +723,14 @@ BUILTIN_MISC_OPERATION(InitializeDefaultActor, "initializeDefaultActor", "", Spe
 /// Destroy the default-actor instance in a default actor object.
 BUILTIN_MISC_OPERATION(DestroyDefaultActor, "destroyDefaultActor", "", Special)
 
+/// Initialize a "proxy" for a distributed remote actor.
+BUILTIN_MISC_OPERATION(InitializeDistributedRemoteActor,
+                       "initializeDistributedRemoteActor", "", Special)
+
+/// Destroy the distributed-actor instance in a "proxy" actor object.
+BUILTIN_MISC_OPERATION(DestroyDistributedActor,
+                       "destroyDistributedActor", "", Special)
+
```

Comment:
Right, so the names i need to polish up a little bit here... but here's the deal:

A `distributed actor` is just the general thing, however a specific instance is going to be either `distributed local actor` or `distributed remote actor`; Initializing a local actor is the same as initializing a plain old DefaultActor, there is no magic about them other than the type-system checks about access to their internals.

A `distributed remote actor` however is very special, as it will allocate _less_ memory and overall is more complicated to initialize, thus it has it's own init.


--- 

You're right that perhaps for ABI it might be better to just have `initializeDistributedRemoteActor` (which will just call initialize default actor pretty much, at least today), `initializeDistributedLocalActor` and `destroyDistributedActor`. We can have just one ABI entry to `destroyDistributedActor` because we know if the actor is remote or not, so we can inside there do the right thing.